macros.xml

```diff
@@ -0,0 +1,86 @@
+<macros>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="3.9">python</requirement>
+            <yield />
+        </requirements>
+    </xml>
+    <xml name="ffindex_single_inputs">
+        <param name="ffdata" type="data" format="ffdata" label="PDB Database" help="Database Data file." />
+        <param name="ffindex" type="data" format="ffindex" label="PDB Database Index" help="Database Index file." />
+    </xml>
+
+    <xml name="ffindex_single_inputs_hhr">
+        <param name="ffdata" type="data" format="ffdata" label="HHR Database" help="Database Data file." />
+        <param name="ffindex" type="data" format="ffindex" label="HHR Database Index" help="Database Index file." />
+    </xml>
+
+    <xml name="citations">
+        <citations>
+            <citation type="doi">10.1021/ci300579r</citation>
+        </citations>
+    </xml>
+    <xml name="logfile">
+        <param name="logfile" type="boolean" truevalue="true" falsevalue="false" checked="false" label="Output log file" />
+    </xml>
+
+    <xml name="pdb_source">
+        <conditional name="pdb_source">
```

Comment:
That here will add a conditional that allows the user to select single files ... but you can also choose from a local ffindex. This local index can be build by me and Nate and we can ship it on CVMFS if we like.